### PR TITLE
backgroundColor of images

### LINKED ISSUE #1255: images should have transparent background
Alle images haben weisse Hintergrundfarbe. Evtl. Zusammenhang mit currentcolor.

Bildschirmfoto vom 2019-08-21 14-19-53

### PR DESCRIPTION
**Pull request by danyball**
fixes #1255

### CODE REVIEW

**Comment by danyball** [comment]
The images now have currentColor set. But color is the foreground --> the SVG sets the "foreground"-color as its fill-color. So we have to manually set the svg-color.

Problems:
- just `color` get through shadowdom, `background` doesnt (parent-wc sets color and child-wc inherited that). 
- cant get computedStyle of parent-wc shadowdom element (where background is set)

File: src/components/20-molecules/policy-features/policy-features-item/index.scss [hunk 1]
```diff
@@ -18,6 +18,22 @@
 
   &__icon {
     max-width: 100%;
+    color: $color-prim-blue-dark-indigo; // default background color for svgs
+
+    // because of fill=currentColor we need to set color to the backgroundColor of the variant
+    &__style {
```
Comment (raphaellueckl) [inline src/components/20-molecules/policy-features/policy-features-item/index.scss]: `style-axa-blue` is not an element, it should be a modifier.

`__icon--axa-blue` for instance. I would leave out the "style" in the property name.

**Comment by danyball** [comment]
> Do we somewhere have a list of all icons to check if they still work and look correctly?

No. We have 300+ images - how to check if they look correct? i thinks thats impossible automatically.

File: src/components/20-molecules/policy-features/policy-features-item/index.scss (hunk 1 above)
Comment (danyball) [reply src/components/20-molecules/policy-features/policy-features-item/index.scss]: Oh, the "__" and "--" have a deeper meaning? Will fix that...

**Comment by raphaellueckl** [comment]
@danyball We could check those icons on one big page where we would load all at the same time. Then we would instantly see, if there is a problem with them. That should be possible, but probably has a big impact on performance.

**Comment by danyball** [comment]
> @danyball We could check those icons on one big page where we would load all at the same time. Then we would instantly see, if there is a problem with them. That should be possible, but probably has a big impact on performance.
> 
> EDIT: Sorry, I accidentially closed it.

I created a feature request for that: https://github.com/axa-ch/patterns-library/issues/1301 (we talked about that idea with marco a few days ago)

**Pull request merged**